test/trento/domain/cluster/cluster_test.exs

```diff
@@ -81,7 +81,7 @@ defmodule Trento.ClusterTest do
       )
     end
 
-    test "should add a host to the cluster" do
+    test "should add a host to the cluster when the cluster has full details" do
```

Comment:
I think the added new info doesn't match the reality of the test, as for the cluster is indifferent whether it had full details or not, as new hosts will be added in all cases